.github/workflows/Tests.yaml

```diff
@@ -506,6 +506,7 @@ jobs:
             install: ON
```

Comment:
[just picking first line in PR]: can you split this into multiple commits? This is hard to review because so many conceptually different things happen all at once.